docs/tutorial/part-four/index.md

```diff
@@ -1,106 +1,91 @@
 ---
-title: Data in Gatsby
+title: Dati in Gatsby
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Welcome to Part Four of the tutorial! Halfway through! Hope things are starting
-to feel pretty comfortable 😀
+Benvenuti nella quarta parte del tutorial! Siete a metà strada! Spero che stiate iniziando a sentirvi a proprio agio 😀
 
-## Recap of the first half of the tutorial
+## Riepilogo della prima metà del tutorial
 
-So far, you've been learning how to use React.js—how powerful it is to be able to
-create your _own_ components to act as custom building blocks for websites.
+Finora, hai imparato ad utilizzare React.js-quanto è utile essere in grado di creare i _propri_ componenti per utilizzarli nei siti web.
 
-You’ve also explored styling components using CSS Modules.
+Hai anche esplorato lo stile dei componenti usando i moduli CSS.
 
-## What's in this tutorial?
+## Cosa c'è in questo tutorial?
 
-In the next four parts of the tutorial (including this one), you'll be diving into the Gatsby data layer, which is a powerful feature of Gatsby that lets you easily build sites from Markdown, WordPress, headless CMSs, and other data sources of all flavors.
+Nelle prossime quattro parti del tutorial (incluso questo), you'll be diving into the Gatsby data layer, which is a powerful feature of Gatsby that lets you easily build sites from Markdown, WordPress, headless CMSs, and other data sources of all flavors.
```

Comment:
this section has both English and Italian version

Reply:
🤦‍♂️